test/converters.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const metatests = require('metatests');
+const { convert: { toJs } } = require('../metaschema');
+
+const simpleSchema = `
+{
+  FirstName: { domain: 'Nomen', required: true },
+  MiddleName: { domain: 'Nomen' },
+}
+`;
+
+const decoratedSchema = `
+{
+  FullName: Include('FullName'),
+  Sex: { domain: 'Sex' },
+  Languages: Many('Language'),
+}
+`;
+
+const allDecoratedSchema = `
+Local({
+  FullName: Include('FullName'),
+  Sex: { domain: 'Sex' },
+  Languages: Many('Language'),
+})
+`;
+
+metatests.testSync('toJs test', test => {
+  test.testSync('must convert simple schema', t => {
+    const expected = `module.exports = ${simpleSchema};`;
+    t.strictSame(toJs(simpleSchema), expected);
+  });
+
+  test.testSync('must convert simple schema (esm)', t => {
+    const expected = `export default ${simpleSchema};`;
+    t.strictSame(toJs(simpleSchema, { useEsm: true }), expected);
+  });
+
+  test.testSync('must convert decorated schema', t => {
+    const expected =
+    `const { decorators } = require('metaschema');
+const { Many, Include } = decorators;
+
+module.exports = ${decoratedSchema};`;
```

Comment:
I do not like multi-line templates

Reply:
IMO it makes it easier to understand/edit this as it is intended to be converted/saved in a file.

Reply:
Ok